src/components/DotNavigation/DotNavigation.meta.tsx

```diff
@@ -0,0 +1,10 @@
+import { DotNavigation } from '.';
+import Registry from '@ui-autotools/registry';
+
+const DotNavigationMetadata = Registry.getComponentMetadata(DotNavigation);
+DotNavigationMetadata.nonReactStrictModeCompliant = true;
+
+DotNavigationMetadata.addSim({
+  title: 'render',
+  props: {},
```

Comment:
We should probably add some props for this test (this tests ssr + hydration and also a11y).
Maybe a length and currentIndex?

Reply:
Done)